openmc/geometry.py

```diff
@@ -257,7 +257,49 @@ def get_all_lattices(self):
                     lattices[cell.fill.id] = cell.fill
 
         return lattices
+    
+    def get_all_surfaces(self):
+        """
+        Return all surfaces used in the geometry
+
+        Returns
+        -------
+        collections.OrderedDict
+            Dictionary mapping surface IDs to :class:`openmc.Surface` instances
 
+        """
+        surfaces = OrderedDict()
+        
+        for cell in self.get_all_cells().values():
+            self._get_surfaces_from_region(surfaces, cell.region)
+        return surfaces
+    
+    def _get_surfaces_from_region(self, surfaces, region):
+        """
+        Recursively find all the surfaces referenced by a region and return them
+        
+        Parameters
+        ----------
+        surfaces: collections.OrderedDict
+            Dictionary mapping surface IDs to :class:`openmc.Surface` instances
+            
+        region: openmc.surface.Region
+            The region of space defined by Surfaces
```

Comment:
There should be a blank line after this. Also, the blank line that is two lines above should be removed.